php/stats.php

```diff
@@ -5,19 +5,13 @@
 include 'helper.php';
 include 'filter.php';
 
-$uid = $_SESSION["uid"];
 $public = "O"; // by default...
-if (!$uid || empty($uid)) {
-    // If not logged in, default to demo mode
-    $uid = 1;
-}
 
-// This applies only when viewing another users flights
```

Comment:
Guess this comment wasn't removed on purpose (still exists in the top10.php code below)

Reply:
^